### PR TITLE
Feature/fun sounds

### PR DESCRIPTION
**Pull request by sparklestheunicorn**
## TL;DR
Fixes a bug where the user can navigate with the menu open without seeing the pages change
Adds more sounds and a system for managing soundscapes

## What
What is affected by this PR?
- [ ] API
- [x] GUI
- [ ] Hardware Integration
- [ ] OS/Deployment
- [ ] Documentation
- [ ] Other (please describe in notes)

## Testing
How was this tested?
- [x] Locally Virtualized
- [ ] Raspberry Pi
- [ ] With Hardware
- [ ] Other (please describe in notes)

## Notes
How was your day?

### CODE REVIEW

File: gui/src/pages/Settings.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useReducer, useContext } from 'react'
-import { ButtonGroup, Button, Icon } from 'semantic-ui-react';
+import { ButtonGroup, Button, Icon, Select } from 'semantic-ui-react';
```
Comment (0phelia36) [inline gui/src/pages/Settings.jsx:2]: This new import isn't used anywhere on the page

File: gui/src/hooks/useAudio.jsx [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
+  [AUDIO_THEMES.SIMPLE]: "wav",
+}
+
+const rootURL = process.env.PUBLIC_URL
+
+export const useAudio = (type) => {
+  const { settings } = useContext(SettingsContext)
+  const theme = settings.audioTheme || AUDIO_THEMES.RAGE
+  const fileUrl = `${rootURL}/audio/${theme}/${type}.${THEME_FORMATS[theme]}`
+  console.log(fileUrl, settings)
```
Comment (0phelia36) [inline gui/src/hooks/useAudio.jsx:27]: This console.log spams the console

File: gui/public/audio/rage/error.mp3
Comment (0phelia36) [inline gui/public/audio/rage/error.mp3:1]: This sounds like it could be a complete sound to me not an error

**Review by 0phelia36 — CHANGES_REQUESTED** [review]

File: gui/src/App.jsx [hunk 3]
```diff
@@ -66,6 +62,10 @@ export function App() {
           <Home status={status} />
         </Route>
 
+        <Route exact path="/">
+          <Home status={status} />
+        </Route>
+
```
[the remark this answers is not in the document]
Comment (sparklestheunicorn) [reply gui/src/App.jsx]: Yes. I had planned to add a "splash" screen to prompt user input before trying to play sounds but instead left the home screen and moved the sounds to the Recipe List page.

**Review by 0phelia36 — APPROVED** [review]

**Pull request merged**